Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN apt-get update && apt-get install -y \
 	protobuf-c-compiler \
 	python-protobuf \
 	&& mkdir -p /usr/src/criu \
-	&& curl -sSL https://github.com/checkpoint-restore/criu/archive/v${CRIU_VERSION}.tar.gz | tar -C /usr/src/criu/ -xz --strip-components=1 \
+	&& for i in {1..5}; do [ $i -gt 1 ] && sleep 1m; curl -sSL https://github.com/checkpoint-restore/criu/archive/v${CRIU_VERSION}.tar.gz | tar -C /usr/src/criu/ -xz --strip-components=1 && break; done \
```

Comment:
`&& break` -> `&& break; (exit $?)`

Reply:
I think we don't need an explicit exit here as RUN directive will stop the hole build on just error returned, and error is returned well:

```[snorch@snorch moby]$ for i in {1..5}; do [ $i -gt 1 ] && sleep 1; false && break; done
[snorch@snorch moby]$ $?
bash: 1: command not found...
[snorch@snorch moby]$ for i in {1..5}; do [ $i -gt 1 ] && sleep 1; true && break; done
[snorch@snorch moby]$ $?
bash: 0: command not found...
```